content/millennium/r4/general-clinical/allergy-intolerance.md

```diff
@@ -76,6 +86,21 @@ Search for AllergyIntolerances that meet supplied query parameters:
 <%= json(:r4_allergy_intolerance_bundle) %>
 <%= disclaimer %>
 
+### Example with RevInclude
+
+### Authorization Types
+
+<%= authorization_types(provider: true, system: true) %>
+
+#### Request
+
+    GET https://fhir-ehr.devcerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/AllergyIntolerance?_id=74771957,S-75479691&_revinclude=Provenance:target
```

Comment:
Update the tenant to "https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d" and in other places as well as relevant.

Reply:
changes addressed